script/windows/14.0/update/WindowsUpdateCheck.php

```diff
@@ -51,22 +54,50 @@ private function BuildKeymanDesktopVersionResponse($tier, $InstalledVersion, $re
       // This gets us upgrades on alpha, beta and stable tiers.
       $tiers = get_object_vars($this->DownloadVersions->windows);
 
-      return $this->CheckVersionResponse($tier, $tiers, $InstalledVersion, $regex);
+      // We will support staying on alpha tier even once a version
+      // hits stable. This is correct, with major upgrade support, as
+      // that'll mean the user will transition to a new version on the
+      // same tier (i.e. always staying bleeding edge). However, for beta
```

Comment:
Technically, during the `beta` cycle, `beta` is more bleeding-edge than `alpha` until the post-sprint merge, right?

Our usual process would have us starting 15.0 alpha at the same time that we create the 14.0 beta branch... and so the update would push alpha users into 15.0 alpha.

Reply:
Correct; they'd receive beta updates periodically -- every two weeks or so as we merge beta changes down to alpha each sprint. The strategy isn't completely perfect but I think I prefer this to shunting them into a stable release each version. I think.